parsers/ENTSOE.py

```diff
@@ -825,50 +827,6 @@ def parse_production(
     )
 
 
-def parse_self_consumption(xml_text: str):
-    """
-    Parses the XML text and returns a dict of datetimes to the total self-consumption
-    value from all sources.
-    Self-consumption is the electricity used by a generation source.
-    This is defined as any consumption source (i.e. outBiddingZone_Domain.mRID)
-    that is not storage, e.g. consumption for B04 (Fossil Gas) is counted as
-    self-consumption, but consumption for B10 (Hydro Pumped Storage) is not.
-
-    In most cases, total self-consumption is reported by ENTSOE as 0,
-    therefore the returned dict only includes datetimes where the value > 0.
-    """
-
-    if not xml_text:
-        return None
-    soup = BeautifulSoup(xml_text, "html.parser")
-    res = {}
-    for timeseries in soup.find_all("timeseries"):
-        is_consumption = (
-            len(timeseries.find_all("outBiddingZone_Domain.mRID".lower())) > 0
-        )
-        if not is_consumption:
-            continue
-        psr_type = str(
-            timeseries.find_all("mktpsrtype")[0].find_all("psrtype")[0].contents[0]
-        )
-        if psr_type in ENTSOE_STORAGE_PARAMETERS:
-            continue
-        resolution = str(timeseries.find_all("resolution")[0].contents[0])
-        datetime_start: arrow.Arrow = arrow.get(
-            timeseries.find_all("start")[0].contents[0]
-        )
-
-        for entry in timeseries.find_all("point"):
-            quantity = float(entry.find_all("quantity")[0].contents[0])
-            if quantity == 0:
-                continue
-            position = int(entry.find_all("position")[0].contents[0])
-            datetime = datetime_from_position(datetime_start, position, resolution)
-            res[datetime] = res[datetime] + quantity if datetime in res else quantity
-
-    return res
```

Comment:
This don't work at all for realtime consumption as far as I can tell and all zones I have checked returned nothing and it is the number 1 cause of warning logs for the ENTSOE zones.

If it's actually required I can add it back but the current implementation made it so we only return 1 datapoint for the consumption function for realtime data.